src/clj/puppetlabs/services/master/master_service.clj

```diff
@@ -17,6 +17,7 @@
    [:CaService initialize-master-ssl! retrieve-ca-cert! retrieve-ca-crl! get-auth-handler]
    [:JRubyPuppetService]
    [:AuthorizationService wrap-with-authorization-check]
+   [:SchedulerService interspaced stop-job]
```

Comment:
Yeah I was wondering about that. We could save the job id, and in the shutdown context for tk also stop the job...but I don't see that for the MasterService? I only see a start context in this file.